settings/Makefile.am

```diff
@@ -34,6 +34,7 @@ xsession_DATA = \
 
 EXTRA_DIST = \
 	dconf-defaults/settings \
+	dconf-defaults/locks \
```

Comment:
Note that by listing an entire directory, rather than the files in it, this will include backup files, `.git` directories (if any exist there, which they shouldn’t), files which aren’t in git, etc.

Possibly better to list the contents of the directory explicitly.

See https://www.gnu.org/software/automake/manual/html_node/Basics-of-Distribution.html#index-EXTRA_005fDIST

I don’t think this nitpick should definitely block merging though.

Reply:
Addressed